src/webviews/webview-side/ipywidgets/common/helper.ts

```diff
@@ -4,12 +4,16 @@
 import { logMessage } from '../../react-common/logger';
 
 const unpgkUrl = 'https://unpkg.com/';
-const jsdelivrUrl = 'https://www.jsdelivr.com/';
+const jsdelivrUrl = 'https://cdn.jsdelivr.net/npm/requirejs@2.3.6/bin/r.min.js';
```

Comment:
The previous url ended up with a redirection and was causing CORs issues, this is a better check (i.e. a better Url)